WooCommerce/Classes/ViewRelated/Blaze/ConfirmPayment/BlazeConfirmPaymentView.swift

```diff
@@ -79,6 +79,9 @@ struct BlazeConfirmPaymentView: View {
             BlazeCampaignCreationLoadingView()
                 .interactiveDismissDisabled()
         }
+        .sheet(isPresented: $viewModel.showAddPaymentSheet) {
+            BlazePaymentMethodsView(viewModel: viewModel.paymentMethodsViewModel!)
```

Comment:
Instead of forced unwrapping, you can use `map` to unwrap the optional value:

```suggestion
            viewModel.paymentMethodsViewModel.map {
                BlazePaymentMethodsView(viewModel: $0)
            }
```

Reply:
Updated in 1f32430929834523c8cb80f3f8bcf8a83d8a6c78